modules/custom/social_language/social_language.services.yml

```diff
@@ -12,3 +12,8 @@ services:
     class: Drupal\social_language\Routing\RouteSubscriber
     tags:
       - { name: event_subscriber }
+  drupal8_url_generator:
```

Comment:
Rename to `drupal_core_url_generator` so that this code will not confuse is if we update to Drupal 9 in 1.5 years time?